tests/fixtures/identity/documents/statuses.js

```diff
@@ -0,0 +1,8 @@
+export const statuses = {
```

Comment:
also rename to constant naming in uppercase

Reply:
done